staging/src/k8s.io/csi-translation-lib/plugins/gce_pd.go

```diff
@@ -202,7 +202,14 @@ func (g *gcePersistentDiskCSITranslator) TranslateInTreeInlineVolumeToCSI(volume
 		partition = strconv.Itoa(int(pdSource.Partition))
 	}
 
-	pv := &v1.PersistentVolume{
+	var am v1.PersistentVolumeAccessMode
+	if pdSource.ReadOnly {
+		am = v1.ReadOnlyMany
```

Comment:
today when an inline volume specifies readOnly, does that cause the attach to be readonly?

Reply:
yeah, specifying `readOnly: true` inline causes the volume to be attached in `mode: READ_ONLY`